resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -104,25 +99,17 @@ export default {
             return Vue.filter('videoTime')(this.hoverTime);
         },
         hoverTimeStyle() {
-            return { 'font': `bold ${this.fontSize}px Sans-Serif`, 'color': '#cccccc', 'bgColor': '#222' };
+            return { 'font': `bold ${this.fontSize}px Sans-Serif`, 'color': '#cccccc' };
         },
         xtext() {
             return this.canvasWidth / 2;
         },
         ytext() {
-            // compute font size in pixel
             return this.canvasHeight + this.fontSizeInPx + (this.hoverTimeBarHeight - this.fontSizeInPx) / 2;
         },
-        fontSizeInPx() {
-            return this.fontSize * (72 / 96);
-        }
```

Comment:
I didn't see that you had to use both `fontSize` and `fontSizeInPx`, sorry. In this case the computed property is indeed a better solution than hard coding both numbers.